uniffi_bindgen/src/bindings/kotlin/templates/Helpers.kt

```diff
@@ -103,7 +103,7 @@ internal inline fun<T> uniffiTraitInterfaceCall(
         writeReturn(makeCall())
     } catch(e: kotlin.Exception) {
         callStatus.code = UNIFFI_CALL_UNEXPECTED_ERROR
-        callStatus.error_buf = {{ Type::String.borrow()|lower_fn }}(e.toString())
+        callStatus.error_buf = {{ Type::String.borrow()|lower_fn }}(e.stackTraceToString())
```

Comment:
Can you wrap this call and the one below in another try..catch to handle the case where `stackTraceToString()` throws?  If that throws, then just store an empty `RustBuffer()` in `error_buf`.

Reply:
I don't think it's really necessary. `stackTraceToString` has no exception declaration, and in fact, it's just a simple utility method in [five lines]( https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/jvm/src/kotlin/util/Exceptions.kt). Kotlin stdlib wraps it for common.